apps/src/sites/studio/pages/googleBlocklyWrapper.js

```diff
@@ -21,6 +21,15 @@ import CdoWorkspaceSvg from '@cdo/apps/blocklyAddons/cdoWorkspaceSvg';
 const BlocklyWrapper = function(blocklyInstance) {
   this.version = BlocklyVersion.GOOGLE;
   this.blockly_ = blocklyInstance;
+
+  /**
+   * If an id is not specified, the Block constructor will randomly generate a 20-character
+   * uid string. In UI tests, we want to be able to reference blocks by their id, which won't
+   * work if the ids are randomly generated. To get around this problem, we can just use a
+   * global counter as the block ids.
+   */
+  this.uidCounter_ = 0;
```

Comment:
Does this only get initialized once when we load a level?

Reply:
Yeah, it's set when we initialize Blockly, which happens on page load